scripts/pre_commit_linter.py

```diff
@@ -251,15 +266,9 @@ def _lint_py_files(config_pylint, files_to_lint, result):
             _MESSAGE_TYPE_SUCCESS, num_py_files, time.time() - start_time))
 
 
-def _pre_commit_linter():
-    """This function is used to check if this script is ran from
-    root directory, node-jscs dependencies are installed
-    and pass JSCS binary path
-    """
-
+# Returns a list of all the files for linting and pattern checks.
```

Comment:
Can you change this comment into a docstring (i.e. """....""" below the function name)?